### ISSUE TITLE
Landing Page search on dev

### ISSUE THREAD

**Issue by paulineobps** [iodepo/OceanBestPractices#231]
Landing page search on dev - whatever search parameter you chose from the dropdown the results page shows the search as labelled **ALL:**

![image](https://user-images.githubusercontent.com/24316483/178583380-874a918a-1af6-4c18-b24a-0db7310360b6.png)

**Comment by paulineobps** [comment]
and the results are totally wrong

**Comment by paulpilone** [comment]
@paulineobps The first search token is generated from the search field and parameter you supplied on the landing (main) search page. Picking a new search field in the search bar like in your screenshot will only be added to the search if you enter a parameter and click enter: e.g.

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/981227/178585684-34f1ae02-fe15-43a1-b4f4-2f94e351b4cb.png">

vs

<img width="860" alt="image" src="https://user-images.githubusercontent.com/981227/178585793-a7232e82-e1a2-46ab-925f-06a216aa10c8.png">

To create the first one - from the main landing page - I selected All Fields and typed "GOOS". From the search results page I changed the drop down to Title and typed "marine".

For the second one - from the main landing page - I selected Endorsed and typed "GOOS".

**Comment by paulineobps** [comment]
sorry Paul not explaining well.  

On the landing page I selected  search field 'Endorsed'  and entered  in the search box GOOS  as per our help text.  I only wanted GOOS Endorsed Practices.

On the result page it displays that my  search was ALL; Endorsed  even though it kept my original Endorsed field search box displayed above  (the 194 results are totally wrong it should be 6 or at least 6 displayed at the top of the list)
![image](https://user-images.githubusercontent.com/24316483/178587562-06dd3627-d5cc-49be-a99d-b10e8670bc62.png).


ANOTHER EXAMPLE
I wanted to search  only in 'Title' field  for 'research infrastructure 'it displays 1342 results! and displays as though I have searched in ALL fields

![image](https://user-images.githubusercontent.com/24316483/178588293-e66cd705-d62c-46a3-95ec-cf1e2e1ddb9d.png)
